DESIGN.md

```diff
@@ -0,0 +1,98 @@
+# Bulk Data Extract (BDE) Processor design
+
+## User requirements
+
+-  The _user interface_ is a read-only PostgreSQL database in AWS RDS, with
+   Azure authentication.
+-  _Accessible_ to all
+   [current BDE users](https://github.com/linz/ds-infra-team/wiki/BDE-users).
+-  Process BDE files
+   _[every day](https://toitutewhenua.atlassian.net/wiki/spaces/GEOD/pages/86418416/BDE+Processor+Product)
+   except Sunday_ into a structure conforming to the legacy BDE Processor DB.
+   The processing should take _much_ less than a day.
```

Comment:
It would be Koordinates code, which is proprietary. Jeremy probably just knows that from having conversations with them and working closely with them in the past. I wasn't aware that table versioning was used by Koordinates. Maybe we could discuss with Rob (their CTO at some point about the possibility of dropping it). They are also moving to a new [backend architecture I believe](https://koordinates.com/blog/sno-our-new-open-source-tool-distributed-data-versioning/) so changes might be possible.